unittests/comprehensive/test_iLQRHalfCheetah.cpp

```diff
@@ -41,195 +40,6 @@ using namespace realtime;
 using namespace trajectory;
 using namespace server;
 
-#ifdef iLQR_TEST
```

Comment:
Sorry, I didn't mean to delete this test! I just meant to uncomment it. I think it'd be good to run this test at every commit.

Reply:
This test is identical to test_iLQR, and the rest of Half cheetah set up should be classified as an example. Therefore, I delete the test case to remove duplication. Sorry for the confusion.